wolfcrypt/test/test.c

```diff
@@ -27521,6 +27521,15 @@ static wc_test_ret_t hpke_test_single(Hpke* hpke)
             ret = WC_TEST_RET_ENC_EC(ret);
     }
 
+    /* NULL argument results in failure */
```

Comment:
In comment please indicate this is a negative test case. I'd like to see additional tests for the other 2 arguments also. Negative test cases need to be at the bottom or top of function. The code bellow assumes a properly setup `hpke` variable.